src/style.css

```diff
@@ -0,0 +1,80 @@
+html {
+  margin: 0;
+  box-sizing: border-box;
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir Heavy;
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 400;
+  font-style: normal;
+}
+
+.search-bar {
+  display: flex;
+  box-sizing: border-box;
+
+  margin-top: 20px;
+  width: 100%;
+  flex-shrink: 0;
+
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  flex-shrink: 0;
+  margin: 24px 1px 24px 25px;
+  /* border: 1px solid transparent; */
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  margin: 14px 0 13px 12px;
+}
+
+.input {
+  margin: 23px 0 23px 62px;
```

Comment:
it is better to use padding on `search-bar` to apply inner geometry rather then margins for input